test/unit/conftest.py

```diff
@@ -320,3 +321,13 @@ def _mock(side_effects, existing_device=None, device=ios_device):
         return device
 
     return _mock
+
+
+@pytest.fixture
+def netmiko_device():
+    # with mock.patch.object(BaseNetmikoDevice, "confirm_is_active") as mock_confirm:
+    #     mock_confirm.return_value = True
+    with mock.patch("netmiko.ConnectHandler") as ch:
+        device = BaseNetmikoDevice("host", "user", "password", "cisco_ios_ssh")
+        device.native = ch
+        yield device
```

Comment:
I'm not seeing why yield here with no changes to what is being yielded. Should this just be a return instead?

Reply:
good catch, I have updated